catkit/hub/cli.py

```diff
@@ -473,6 +485,15 @@ def connect(user):
     show_default=True,
     help="Store intermediate filetype as traj"
     "instead of json files")
+@click.option(
+    '-u', '--use-cache',
+    type=bool,
+    is_flag=True,
+    default=False,
+    show_default=True,
+    help="When set the script will structures"
```

Comment:
Something's wrong with this sentence. Do you mean "When set, structures will be cached..."?

Reply:
Awkward english. Is fixed now.